securedrop/sass/journalist.sass

```diff
@@ -101,3 +101,15 @@ button.small-danger, a.btn.small-danger, .btn.small-danger
 
 .journalist-reply__input
   max-width: 700px
+
+.journalist-username__notes
+  font: 12px Sans-Serif
+  font-weight: normal
+  letter-spacing: 0.05px
+
+.code_modified
```

Comment:
Firstly, you need to put this code [here](https://github.com/freedomofpress/securedrop/blob/develop/securedrop/sass/modules/_code.sass). Secondly, what is the benefit of removing the `padding` and the `border-bottom`? @ninaviz what do you think about this design vs the one we have for `.code` ?

Reply:
@SaptakS I removed padding for highlighted password part as it was taking more space on the upper side but later after @heartsucker said that font-size is too small so I increased the font size and added padding with suitable pixels.I will push PR after modifying it again.